apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,14 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimFrame = document.createElement('iframe');
```

Comment:
also wondering if we could not give the instance id using crossframe javascript (direct calls as it's possible because we're in the same domain). So we could have a static html and benefit from cache and loads it statically in index.html ?